content/about/center-of-excellence/index.md

```diff
@@ -30,27 +30,26 @@ members:
     - lipsitch
 
 sisters:
-- name: CDC's Influence Division
-  description: Our funders and collaborators at the [CDC's Influenza Division](https://www.cdc.gov/ncird/flu.html), specially the Influenza Applied Research & Modeling team at the [Epidemiology and Prevention branch](https://www.cdc.gov/hiv/dhap/eb/index.html).
+- name: CDC's Influenza Division
+  description: Our funders and collaborators at the [CDC's Influenza Division](https://www.cdc.gov/ncird/flu.html), in particular the Influenza Applied Research and Modeling team at the [Epidemiology branch](https://www.cdc.gov/hiv/dhap/eb/index.html).
   link: https://www.cdc.gov/ncird/flu.html
-  img: logos/cdc.png
 - name: University of Massachusetts Amherst
-  description: Our sister organization, the [Influenza Forecasting Center of Excellence at UMass Amherst](https://reichlab.io/).
+  description: Our sister organization, the [Influenza Forecasting Center of Excellence](https://reichlab.io/) at University of Massachusetts Amherst.
   link: https://reichlab.io/
-  img: logos/reichlab.png
 
 supporters:
-- Allegheny County Dept of Health
+- Allegheny County Department of Health
 - Change Healthcare
-- DTRA
+- Defense Thread Reduction Agency
 - Facebook
+- Google.org
 - Kaiser Permanente Washington Health Research Institute
-- Massachusetts Dept of Public Health
+- Massachusetts Department of Public Health
 - Microsoft Azure
-- Pennsylvania Dept. of Health
-- Quidel Inc.
-- UnitedHealth / Optum
-- UPMC
+- Pennsylvania Department of Health
+- Quidel
```

Comment:
Re the Quidel Inc. -> Quidel change -- we currently have "Quidel Inc." as the official name of this source in the covidcast dataset metadata. Should I change the name there to "Quidel" as well?

Reply:
I would vote for "Quidel Inc.", because (1) this is how Quidel usually refers to itself; and (2) It is not a well known name so the "Inc." helps identify it as a corporation.  But either way is acceptable.